hack/update-codegen.sh

```diff
@@ -20,7 +20,7 @@ if  ! ( command -v controller-gen > /dev/null ); then
   fi
 fi
 
-"${SCRIPT_ROOT}/vendor/k8s.io/code-generator/generate-groups.sh" deepcopy \
+bash "${SCRIPT_ROOT}/vendor/k8s.io/code-generator/generate-groups.sh" deepcopy \
```

Comment:
why?

Reply:
because the shell script isn't executable.